src/scss/settings/_variables.scss

```diff
@@ -117,9 +117,9 @@ $zIndex: (
 
 // Path vars
 // ==========================================================================
-$img-path                  : '/assets/dist/img'; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
+$img-path                  : './dist/img'; // Usage: background: url('#{$img-path}/img.jpg') 0 0 no-repeat;
```

Comment:
Created this issue https://github.com/justeat/gulp-build-fozzie/issues/61 to cover the asset sharing